src/content/en/tools/lighthouse/audits/blocking-links.md

```diff
@@ -0,0 +1,61 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site Does Not Use Link Tags That Delay First Paint" Lighthouse audit.
+
+{# wf_updated_on: 2016-12-01 #}
+{# wf_published_on: 2016-12-01 #}
+
+# Site Does Not Use Link Tags That Delay First Paint  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Fast page loads result in higher user engagement, more pageviews, and
+improved conversion.
+
+You can improve your page load speed by inlining resources that are required
+for first paint, and deferring those that aren't.
+
+## How to pass the audit {: #how }
+
+In your report, Lighthouse lists all of the render-blocking links that it has
+detected. The goal is to reduce this number.
+
+As mentioned in [How the audit is implemented](#implementation), Lighthouse
+flags three types of render-blocking links: scripts, stylesheets, and HTML
```

Comment:
Ah, I see. I might want to split these up in the future, but this is fine for now. I'll change the title.